oojs.js

```diff
@@ -1,4 +1,27 @@
 // Define a Playlist class. It should have...
+class Playlist {
+  constructor(title){
```

Comment:
The songs array should have been initialized by some parameter to the constructor:
```js
  constructor(title, songs){
    // tries to use the songs parameter, if `undefined` will assign an empty array instead
    this.songs = songs || [];
```